clouddriver-kubernetes/src/test/groovy/com/netflix/spinnaker/clouddriver/kubernetes/provider/view/KubernetesJobProviderSpec.groovy

```diff
@@ -108,10 +108,65 @@ class KubernetesJobProviderSpec extends Specification {
     }
 
     when:
-    def provider = new KubernetesJobProvider(mockAccountCredentialsProvider, mockManifestProvider)
+    def provider = new KubernetesJobProvider(mockAccountCredentialsProvider, mockManifestProvider, true)
```

Comment:
What's the extra `true` for?

Reply:
The [KubernetesJobProvider constructor](https://github.com/spinnaker/clouddriver/blob/master/clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/provider/view/KubernetesJobProvider.java#L52-L55) has three parameters - `true` is the default value for the `detailedPodStatus` parameter.

Reply:
Aaah, thanks.